tests/RocksDBIteratorTest.cpp

```diff
@@ -0,0 +1,118 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+
+#include "iterlib/RocksDBIterator.h"
+
+#include <boost/filesystem.hpp>
+#include <folly/Singleton.h>
+#include <gtest/gtest.h>
+#include <memory>
+
+#include "iterlib/LimitIterator.h"
+#include "rocksdb/comparator.h"
+#include "rocksdb/db.h"
+#include "rocksdb/options.h"
+#include "rocksdb/table.h"
+
+using namespace rocksdb;
+std::string db_path;
+
+class RocksDBIteratorTest : public ::testing::Test {
+ public:
+  RocksDBIteratorTest() {}
+
+  rocksdb::DB* getDB() { return db_; }
+
+  rocksdb::Status Put(const Slice& k, const Slice& v,
+                      WriteOptions wo = WriteOptions()) {
```

Comment:
Did you mean ConstructorInitializerAllOnOneLineOrOnePerLine=true in clang-format? I tried that, but clang-format reverts it back to the formatting above. Perhaps it applies only to constructors?